### PR TITLE
DDGSyntaxCheatSheet: Improve triggering

### PR DESCRIPTION
**Pull request by sahildua2305**
###### What does your Pull Request do (check all that apply)?

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [x] Enhancement: **`DDGSyntaxCheatSheet: Improve triggering`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes
 - Removed unnecessary triggers generating warnings because `syntax` is already defined in `code` template
 - Added new triggers

---

Instant Answer Page: https://duck.co/ia/view/duckduckgo_syntax_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @gautamkrishnar

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by gaulrobe** [comment]
@sahildua2305 careful here, https://github.com/sahildua2305/zeroclickinfo-goodies/blob/master/share/goodie/cheat_sheets/json/duckduckgo.json is also a valid DDG cheat sheet. We'll want triggers that differentiate between the two.

**Comment by sahildua2305** [comment]
@gaulrobe But we can't just get away with wrong aliases (which generate warnings on testing), right?
What would you suggest?

**Comment by gaulrobe** [comment]
@sahildua2305 maybe I'm not understanding something myself. What warnings are you talking about?

**Comment by sahildua2305** [comment]
Here @gaulrobe 
https://travis-ci.org/duckduckgo/zeroclickinfo-goodies/builds/138474324#L684

**Comment by gaulrobe** [comment]
@sahildua2305 I wonder if that is a sign that this cheat sheet should be `code` instead of `reference`?

**Comment by ManrajGrover** [comment]
I'm not sure if its correct to submit individual PR for every warning generated in Travis Build. I planned to fix them myself but then @122131 has already asked it [here](https://github.com/duckduckgo/zeroclickinfo-goodies/pull/3268#issuecomment-226720840)

**Comment by sahildua2305** [comment]
@gaulrobe May be! Changing to `code` will definitely solve the purpose. However, I thought the triggering can be improved. Let me know what you think.

**Comment by sahildua2305** [comment]
@ManrajGrover If you check the recent Travis Build, there are a lot of goodies generating warnings. I have submitted PRs for those for which I felt they needed additional changes (or not, may be).

I am not sure what's wrong with making changes that make the IAs (or their triggers/testing) better. Not sure if there's someone out there counting our PRs :smile_cat: 
cc @moollaza @zekiel

**Comment by sahildua2305** [comment]
Sorry @122131 I didn't know you had already asked about this before me.
Upto you now, go for it :smiley:

**Comment by ManrajGrover** [comment]
@sahildua2305 Its just that a single PR would be easy to track rather than individual PRs. Sure each PR can be dealt individually but would just make review process a little more tedious. 

Nothing wrong in making IAs better. Infact, that is why we all are here. :blush: To make web a better and free place. But some points should be put into consideration.

**Comment by sahildua2305** [comment]
@ManrajGrover Sure! I agree with what you are saying.
I created two PRs instead of one, considering the fact that I was unaware of what @122131 had mentioned in that PR, neither did that PR give any such hint from its title. Not a big deal, bro! :smiley_cat: 

But I will still need inputs from you, @moollaza, @zekiel and @tagawa on this issue raised.

**Comment by gaulrobe** [comment]
@sahildua2305 @ManrajGrover 

To me there seems to be a difference in the white space fixes that @122131 alluded to fixing in #3268 and fixes such as this one where triggers are being changed and/or templates being changed. White space changes will _probably_ not need maintainer input and can _probably_ be in one PR. Changes that affect triggering and possibly design will need maintainer input. My two cents :smile:

**Comment by 122131** [comment]
@gaulrobe @sahildua2305 @ManrajGrover 
If you want I can start by opening a single PR for trailing whitespaces only, or I can just edit #3268.

File: share/goodie/cheat_sheets/json/duckduckgo-syntax.json [hunk 1]
```diff
@@ -8,7 +8,9 @@
     "sourceUrl": "https://duck.co/help/results/syntax"
   },
   "aliases": [
-    "ddg syntax","duck duck go syntax"
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/duckduckgo-syntax.json]: Please separate this into two lines

Comment (sahildua2305) [inline share/goodie/cheat_sheets/json/duckduckgo-syntax.json]: That's the deleted line @gautamkrishnar

Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/duckduckgo-syntax.json]: @sahildua2305  i mean
`
 "ddg syntax",
"duck duck go syntax"
`

File: share/goodie/cheat_sheets/json/duckduckgo-syntax.json [hunk 2]
```diff
@@ -8,7 +8,9 @@
...
+    "ddg search",
+    "duck duck go search",
+    "duckduckgo search"
```
Comment (sahildua2305) [inline share/goodie/cheat_sheets/json/duckduckgo-syntax.json]: That's what I did here, @gautamkrishnar 
I am not sure I understood what you meant.

Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/duckduckgo-syntax.json]: Okay sorry for the confusion

**Comment by gautamkrishnar** [comment]
I think this one is good to merge :+1:  
CC: @GuiltyDolphin @moollaza

**Comment by gaulrobe** [comment]
@122131 go ahead and make a new PR for whitespace changes. Thanks!

**Pull request merged**